tests/testthat/test-read_package.R

```diff
@@ -131,18 +129,27 @@ test_that("read_package() allows descriptor at absolute or relative parent
            path", {
   relative_path <- "../testthat/data/valid_minimal.json"
   expect_true(
-    check_package(suppressMessages(read_package(relative_path)))
+    check_package(read_package(relative_path, quiet = TRUE))
   )
   absolute_path <- normalizePath("data/valid_minimal.json")
   expect_true(
-    check_package(suppressMessages(read_package(absolute_path)))
+    check_package(read_package(absolute_path, quiet = TRUE))
   )
 })
 
 test_that("read_package() allows YAML descriptor", {
   expect_true(
     check_package(
-      suppressMessages(read_package(test_path("data/valid_minimal.yml")))
+      read_package(test_path("data/valid_minimal.yml"), quiet = TRUE)
     )
   )
 })
+
+test_that(paste(
```

Comment:
I would not use paste for test descriptions, but simplify to:

> read_package() can silence message with quiet=TRUE

Reply:
done